docs/cloud/cloud-authentication/cloud-auth-rename-key.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Update Key Description
+layout: default
+parent: Manage API Keys
+grand_parent: Cloud
+nav_order: 4
+---
+
+# How do I update an API key's description in FeatureBase Cloud?
+{: .no_toc }
+
+Learn how to update a FeatureBase Cloud API key's description.
+
+{% include page-toc.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Create an API key](/docs/cloud/cloud-authentication/cloud-auth-create-key/)
+
+## Update an API key's description
+
+* Click **Configuration** > **Manage API keys**.
+<ul>
```

Comment:
Not sure why you've hardcoded HTML bullets here? Unless there's something weird going on with the system??

Reply:
I had copied that from somewhere else because it had the code to display the kebab menu symbol. No idea why the bullets were hardcoded. It's fixed now.

Reply:
I also changed it in the place I copied it from 👍